src/styles/util.scss

```diff
@@ -89,3 +89,11 @@
 .u-color-text--muted {
   color: var.$color-text--light;
 }
+
+// TODO: The sidebar should really apply vertical spacing to its immediate children,
+// but until that happens, this utility class allows a container to assert
+// a bottom margin to keep things in the sidebar evently spaced. The spacing
+// unit should also be updated.
+.u-sidebar-container {
+  margin-bottom: 1em;
```

Comment:
What are the emerging rules around when to use em vs rem vs px spacing? I think you did write about this in a previous PR comment but I think it would be helpful to write that up as documentation somewhere that is easier to find.

Reply:
The spacing for these elements already was `em`. We're moving towards `rem` but in this case we've got a significantly-smaller-than-1-rem local font size, so `1rem` here looks too gappy. I'd summarize the current state as:

* Definitely not `px` for spacing in any new or updated style rules
* Moving towards `rem`
* Finger-in-dike `em` usage still where migrating to `rem` right this second might cause visually-noticeable changes

I'm trying for a balance between "really clean and clear" (e.g. "The convention is for `rem`s in a 0-9 scale as defined in the shared package") and "easily usable in-situ and organic (e.g. "Don't use `px` anymore, use `rem` when you can, but `em` is OK during this transition time"). But perhaps it's falling too far on the loosey-goosey side? Always a bit of a balance when working on so many bits of the design system at once...